demo/agent_monitor/tenant_agent_monitor.py

```diff
@@ -148,18 +148,18 @@ def initthread():
                                     break
                                 logger.debug("init-output: %s" % line.strip())
 
-                        t = threading.Thread(target=initthread)
-                        t.start()
+                        thread = threading.Thread(target=initthread)
+                        thread.start()
 
                     keylime.web_util.echo_json_response(self, 200, "Success", json_body)
                     logger.info("POST returning 200 response for Agent Monitor connection as " + agent_id)
             else:
                 keylime.web_util.echo_json_response(self, 400, "uri not supported")
                 logger.warning("POST returning 400 response. uri not supported")
-        except Exception as e:
-            keylime.web_util.echo_json_response(self, 400, "Exception error: %s" % e)
-            logger.warning("POST returning 400 response. Exception error: %s" % e)
-            logger.exception(e)
+        except Exception as err:
```

Comment:
for my curiosity, what was the complain here, a single char variable name? I am reading the [C0103](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/invalid-name.html) documentation but I do not see a good candidate.

i, e, x, y are very used as local variables, if pylint complain if somehow a bit unfair